### ISSUE TITLE
In ference of MB-MelGAN, it  generated slow audio than raw?

### ISSUE THREAD

**Issue by ymzlygw** [TensorSpeech/TensorFlowTTS#680]
Hi,  I trained a mb-melgan model with 16kHz dataset. But after inference, I found that generated audio speed is slow than raw audio. 

I just change the config file's two params when training:  **sampling_rate-> 16000,  hop_size-> 200.**

Is there anything wrong? 
**Here are the generated and raw samples:** 
Because they have same length but different speed, so the generated wav sounds cut at the end.
https://drive.google.com/drive/folders/1rWNn4J8vWam4CvzH68rw0U9CNjuPvELt?usp=sharing

**Comment by dathudeptrai** [comment]
@ymzlygw the mel input for Melgan is ground-truth?

**Comment by ymzlygw** [comment]
> @ymzlygw the mel input for Melgan is ground-truth?

Not ground-truth, it uses the mel-output generated directly by tacotron model. and then normalize it to (-1,1)

**Comment by dathudeptrai** [comment]
@ymzlygw then the problem come from tacotron2 :D

**Comment by Moonmore** [comment]
you solved this problem?
